relay/kusama/src/lib.rs

```diff
@@ -1626,9 +1606,6 @@ construct_runtime! {
 		Auctions: auctions::{Pallet, Call, Storage, Event<T>} = 72,
 		Crowdloan: crowdloan::{Pallet, Call, Storage, Event<T>} = 73,
 
-		// State trie migration pallet, only temporary.
-		StateTrieMigration: pallet_state_trie_migration = 98,
```

Comment:
We probably need to write some migration to remove all the old pallet storage items? There already exists the `RemovePallet` migration that we can use for this.

Reply:
yes I will try to use the code we ran on westend